src/bundles/analytics.js

```diff
@@ -442,8 +453,9 @@ const createAnalyticsBundle = ({
       state = state || {
         lastEnabledAt: 0,
         lastDisabledAt: 0,
+        showAnalyticsComponents: DISABLE_ALL_ANALYTICS, // hide related UI  for now, see https://github.com/ipfs/ipfs-webui/issues/2198
         showAnalyticsBanner: false,
-        optedOut: false,
+        optedOut: !DISABLE_ALL_ANALYTICS, // disable analytics by default for now, see https://github.com/ipfs/ipfs-webui/issues/2198
```

Comment:
```suggestion
        optedOut: DISABLE_ALL_ANALYTICS, // disable analytics by default for now, see https://github.com/ipfs/ipfs-webui/issues/2198
```

Reply:
don't we want to set this opted out to true for now? if not, we should probably leave as `false`

Reply:
Yeah my bad, I negated wrong line. Fixed in 6f36572b12ca89db88260d85d3f9c85346f067db + updated tests to confirm we hide UI and do no tracking by default.